src/foundry/foundry.js/ray.d.ts

```diff
@@ -34,25 +66,35 @@ declare class Ray {
   slope: number;
 
   /**
-   * The normalized angle of the ray in radians on the range (-PI, PI)
+   * The cached angle, computed lazily in Ray#angle
+   * @internal
+   */
+  protected _angle: number;
+
+  /**
+   * The cached distance, computed lazily in Ray#distance
+   * @internal
    */
-  angle: number;
+  protected _distance: number;
```

Comment:
```suggestion
  protected _distance: number | undefined;
```

Also default value `undefined` should be documented.